src/main/java/com/dev/cinema/dto/CinemaHallDto.java

```diff
@@ -0,0 +1,10 @@
+package com.dev.cinema.dto;
+
+import lombok.Data;
+
+@Data
+public class CinemaHallDto {
```

Comment:
let's remove this dto - we don't use it at the moment

Reply:
Ok